packages/swap/src/utils/getPoolQuote.ts

```diff
@@ -114,7 +118,7 @@ export default async function getPoolQuote({
       destAsset,
       boosted: Boolean(boostFeeBps),
     }),
-    estimatedPrice: getPrice(swapInputAmount, srcAsset, egressAmount, destAsset),
+    estimatedPrice: getPrice(swapInputAmount, srcAsset, swapOutputAmount, destAsset),
```

Comment:
```suggestion
    estimatedPrice: getPrice(depositAmount, srcAsset, egressAmount, destAsset),
```
not holding a strong opinion here, I just feel like the user doesn't care about ingress/egress fees. they just care about how much they sent and how much they got and that would be the effective exchange rate aka price for them

Reply:
ah I see, so you're doing this to make it aligned with fok params. that's reasonable

Reply:
if you calculate `quote.estimatedPrice * 0.99` right now, you will accept a slightly higher slippage than 1% (or a significantly higher slippage if the ingress/egress fees are high relative to the quoted amount)

Reply:
maybe we should use a more explicit name like `estimatedAmmPrice`/`estimatedFillPrice` and deprecate this one to make it more explicit